src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml

```diff
@@ -31,10 +32,47 @@
       ClipToBounds="True"
       Focusable="False"
       Visibility="{Binding Path=IsReadmeReady, Mode=OneWay, Converter={StaticResource BooleanToVisibilityConverter}}" />
-    <TextBlock
-      TextWrapping="Wrap"
-      Text="{x:Static nuget:Resources.Text_Loading}"
-      Visibility="{Binding Path=IsBusy, Mode=OneWay, Converter={StaticResource BooleanToVisibilityConverter}}" />
+    <Grid
+      HorizontalAlignment="Center"
+      VerticalAlignment="Center"
+      Visibility="{Binding Path=IsBusy, Mode=OneWay, Converter={StaticResource BooleanToVisibilityConverter}}" >
+      <Grid.ColumnDefinitions>
+        <ColumnDefinition Width="Auto"/>
+        <ColumnDefinition Width="Auto"/>
+      </Grid.ColumnDefinitions>
+      <nuget:Spinner
+        Grid.Column="0"
+        Height="20"
+        Width="20"/>
+      <TextBlock
+        Padding="10,0,0,0"
+        Grid.Column="1"
+        TextWrapping="Wrap"
+        Text="{x:Static nuget:Resources.Text_LoadingReadme}" />
+      <!--
+        Special TextBlock to report loading status to assistive technologies (narrator).
+        Code behind changes to the Text property will trigger a narrator event.
+      -->
+      <vs:LiveTextBlock
+        x:Name="_ltbLoading"
```

Comment:
The name is not needed unless you want to reference it in codebehind, which we try to avoid as much as possible.